pkg/metrics/observers/skipper.go

```diff
@@ -0,0 +1,110 @@
+package observers
+
+import (
+	"fmt"
+	"regexp"
+	"time"
+
+	flaggerv1 "github.com/weaveworks/flagger/pkg/apis/flagger/v1beta1"
+	"github.com/weaveworks/flagger/pkg/metrics/providers"
+)
+
+var skipperQueries = map[string]string{
+	"request-success-rate": `
+	{{- $route := printf "kube_%s__%s.*__%s" namespace ingress service }}
+	sum(
+		rate(
+			skipper_response_duration_seconds_bucket{
+				namespace="{{ namespace }}",
+				route=~"{{ $route }}",
+				code!~"5.*",
```

Comment:
code!~"5..",

Reply:
:bug:

Reply:
good find - curious that other observers all use `5.*`

adjusted for skipper observer